prep-javascript-primitives-booleans-undefined-null/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Primitive Data Types - Boolean Undefined Null</title>
+</head>
+
+<body>
+  <script>
+    //Your code here
+    var favoriteItem = "Car",
+    price = 3.5
+    isLaptop = Boolean(favoriteItem == "laptop");
```

Comment:
You are missing the comma for the line above, so this variable is not being declared correctly. In general, I recommend using `var` for every variable you declare, rather than connecting multiple with commas 👍

Reply:
gotcha, I'll var and terminate for the rest of class :)